ple/games/donkeykong/README.md

```diff
@@ -0,0 +1,9 @@
+# DonkeyKong-Pygame
```

Comment:
Could you put this into the docs? Similar to how the other games are setup? [Example](http://pygame-learning-environment.readthedocs.io/en/latest/user/games/catcher.html) and [source](https://raw.githubusercontent.com/ntasfi/PyGame-Learning-Environment/master/docs/user/games/catcher.rst). I can make the gif of the game so dont worry about that part.